README.md

```diff
@@ -2,31 +2,187 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Extend `RuntimeException` to create a new unchecked exception class
+- Extend `Exception` to create a new checked exception class
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+We will create two new exception classes `NegativeNumberException`
+and `LongStringException` to demonstrate unchecked and checked exceptions.
 
-## Topic 1
+![new exceptions uml](https://curriculum-content.s3.amazonaws.com/6677/pillars/new_exceptions.png)
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+## Extending `RuntimeException`
 
-## Topic 2
+Let's create a method that takes an integer as a parameter and prints a countdown,
+for example:
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+```text
+4
+3
+2
+1
+0
+Blastoff!
+```
 
-## Conclusion
+We can't do a countdown to 0 if the parameter is negative.  We'll create a new
+class `NegativeNumberException` that extends `RuntimeException`, which
+means it represents an unchecked exception.  The constructor will take
+a string to display when the inherited `getMessage()` method is called.
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+```java
+public class NegativeNumberException extends RuntimeException {
 
-## Resources
+    public NegativeNumberException(String message) {
+        super(message);
+    }
 
-- [Resource Link 1](example.com)
-- [Resource Link 2](example.com)
+}
+```
+
+The class `UncheckedExceptionMain` shown below has the method `countDown(int start)`,
+which throws a `NegativeNumberException` if the parameter value is negative.
+Since the exception is unchecked, the `countDown(int start)` method need not declare it throws the exception.
```

Comment:
"... the `countDown(int start)` method need not declare it throws the exception."

Consider rewording to:

"... the `countDown(int start)` method does not need to declare that it throws an exception."

Reply:
reworded